src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Appearance/WindowBackdrop.cs

```diff
@@ -287,6 +274,20 @@ private static bool EnableGlassFrame(IntPtr hWnd, WindowBackdropType backdropTyp
         return true;
     }
 
+    private static void ApplyBackdropOnLoad(object sender, RoutedEventArgs e, WindowBackdropType backdropType)
+    {
+        IntPtr windowHandle =
+            new WindowInteropHelper(sender as System.Windows.Window ?? null)?.Handle ?? IntPtr.Zero;
+
+        if (windowHandle == IntPtr.Zero)
+        {
+            return;
+        }
+
+        ApplyBackdrop(windowHandle, backdropType);
+        (sender as System.Windows.Window).Loaded -= (sender, e) => ApplyBackdropOnLoad(sender, e, backdropType);
```

Comment:
Can you ask pankaj or ashish, if this way is right ? I mean the pattern is right, but since we are using lambda functions, esentially it is possible that two different lambda functions are generated and the first one is not unhooked.

This is a possibility that crossed my mind. I don't know the internals of lambda functions. If somewhere else we have worked in this way, ( i.e. unhooking via lambda methods ) then this is good to go